fast64_internal/oot/oot_object.py

```diff
@@ -0,0 +1,40 @@
+from ..utility import ootGetSceneOrRoomHeader
+
+from .data.oot_data import OoT_Data
+from .oot_scene_room import OOTRoom
+from bpy.types import Object
+
+
+def addMissingObjectToProp(roomObj: Object, headerIndex: int, objectKey: str):
+    """Add the missing object to the room empty object OoT object list"""
+    if roomObj is not None:
+        roomProp = ootGetSceneOrRoomHeader(roomObj, headerIndex, True)
+        if roomProp is not None:
+            collection = roomProp.objectList
+            collection.add()
+            collection.move(len(collection) - 1, (headerIndex + 1))
+            collection[-1].objectKey = objectKey
+
+
+def addMissingObjectsToList(roomObj: Object, room: OOTRoom, ootData: OoT_Data, headerIndex: int):
```

Comment:
I think I'd rename this function to `addMissingObjectsToRoomHeader`
and `addRoomHeadersObjects` to `addMissingObjectsToAllRoomHeaders`
?